quilt/tools/build.py

```diff
@@ -198,19 +199,26 @@ def _generate_contents(dir_path):
             try:
                 safename = _pythonize_name(nodename)
                 if safename in contents:
-                    print("Warning: duplicate name %r in %s." % (safename, dir_path))
-                    continue
+                    if ext:
+                        safename = "{name}_{ext}".format(name=safename, ext=ext)
```

Comment:
do we know that they have same parent directory? if so then OS should enforce unique naming. what am i missing?